__test__/containers/UserMenu.test.js

```diff
@@ -8,35 +8,43 @@ import { StyleSheetTestUtils } from "aphrodite";
 import { UserMenu } from "../../src/containers/UserMenu";
 import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
 
+function getData(isSuperAdmin = "false") {
```

Comment:
isSuperAdmin should be a boolean, not a string, so remove the parens?

Reply:
yep!